util/config/schema.py

```diff
@@ -1202,6 +1202,16 @@
             "description": "Enables system default quota reject byte allowance for all organizations",
             "x-example": False,
         },
+        "QUOTA_TOTAL_DELAY_SECONDS": {
+            "type": "int",
+            "description": "The time to delay the Quota backfill operation. Must be set longer than the time required to complete the deployment.",
+            "x-example": 30,
+        },
+        "WHITELISTED_QUOTA_ORGS": {
```

Comment:
This parameter is not used in this PR

Reply:
Correct, that was left over from testing. I removed it.